pkg/container/resolver_test.go

```diff
@@ -41,6 +41,7 @@ func TestResolver(t *testing.T) {
 			common.ToPtr(""),
 			common.ToPtr(false),
 			false,
+			nil,
```

Comment:
Since we're cleaning things / tidying up, let's initialise `SourceSpec{}` with named parameters here (and below).